packages/api/get-stops.ts

```diff
@@ -7,7 +7,7 @@ import { getPolygon } from "./utils";
 const MAX_ATCO_CODES = process.env.MAX_ATCO_CODES || "5";
 const MAX_NAPTAN_CODES = process.env.MAX_NAPTAN_CODES || "5";
 const MAX_ADMIN_AREA_CODES = process.env.MAX_ADMIN_AREA_CODES || "5";
-const MAX_POLYGON_AREA = process.env.MAX_POLYGON_AREA ? Number(process.env.MAX_POLYGON_AREA) : 100000000;
+const MAX_POLYGON_AREA = process.env.MAX_POLYGON_AREA ? Number(process.env.MAX_POLYGON_AREA) : 36;
```

Comment:
Might be worth making this and the env vars `MAX_POLYGON_AREA_IN_KM2` to make it clear

Reply:
done